cpp/ql/src/codeql-suites/cpp-lgtm-full.qls

```diff
@@ -0,0 +1,14 @@
+- description: Standard LGTM queries for C/C++, including ones not displayed by default
+- qlpack: codeql-cpp
+- apply: lgtm-selectors.yml
+  from: codeql-suite-helpers
+# These queries are infeasible to compute on large projects:
+- exclude:
+    dxquery path:
+      - Security/CWE/CWE-497/ExposedSystemData.ql
+      - Critical/DescriptorMayNotBeClosed.ql
+      - Critical/DescriptorNeverClosed.ql
+      - Critical/FileMayNotBeClosed.ql
+      - Critical/FileNeverClosed.ql
+      - Critical/MemoryMayNotBeFreed.ql
+      - Critical/MemoryNeverFreed.ql
```

Comment:
@jbj: These exclusions are from `generate-lgtm-suites.py` but the actual queries are already excluded, by virtue of having no precision tag. Are they still needed?

Reply:
I just checked the first one, and it has `@precision medium`: https://github.com/Semmle/ql/blob/1c79ec550eb2193708178029fa787a6c8defd07e/cpp/ql/src/Security/CWE/CWE-497/ExposedSystemData.ql#L8

That means it would be run but not displayed on LGTM.

Reply:
Okay, @hmakholm's comment applies only to the 6 queries highlighted.

I'd prefer to keep them in this file. They were added defensively in internal PR 22075 to ensure that we're free to change `@precision` in the future without accidentally putting these queries in the suite.